templates/buildkite-elastic.yml

```diff
@@ -126,6 +127,11 @@ Parameters:
     Default: t2.nano
     MinLength: 1
 
+  EBSOptimized:
+    Description: Optional - Enabled EBSOptimized for the instance
```

Comment:
@ans0600 are you able to follow up with Amazon and enquire? I'm reluctant to make this change without some more feedback on how it should work.